src/Lib.P2P.Tests/SecureCommunication/Psk1StreamTest.cs

```diff
@@ -0,0 +1,97 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System;
+using System.IO;
+using Lib.P2P.Cryptography;
+using Lib.P2P.SecureCommunication;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Lib.P2P.Tests.SecureCommunication
+{
+    [TestClass]
+    public class Psk1StreamTest
+    {
+        [TestMethod]
+        [ExpectedException(typeof(Exception))]
+        public void BadKeyLength()
+        {
+            var psk = new PreSharedKey();
+            var _ = new Psk1Stream(Stream.Null, psk);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable '_'.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)